src/api/app/models/package.rb

```diff
@@ -1318,6 +1318,14 @@ def bugowner_emails
     (relationships.bugowners_with_email.pluck(:email) + project.bugowner_emails).uniq
   end
 
+  # Returns an ActiveRecord::Relation with all BsRequest that the package is somehow involved in
+  def bs_requests
+    BsRequest.order('number DESC').includes(:bs_request_actions, :comments, :reviews, :labels)
+             .where(bs_request_actions: { source_package_id: id })
+             .or(BsRequest.includes(:bs_request_actions, :comments, :reviews, :labels).where(bs_request_actions: { target_package_id: id }))
+             .or(BsRequest.includes(:bs_request_actions, :comments, :reviews, :labels).where(reviews: { by_package: name }))
```

Comment:
```suggestion
             .or(BsRequest.includes(:bs_request_actions, :comments, :reviews, :labels).where(reviews: { by_package: name, by_project: project.name }))
```
If I'm not wrong, when we ask for a review made "by a package", we also add the "by a project". See: https://github.com/openSUSE/open-build-service/blob/b741ef6a004afcfc4cfd7086e8db5ebb76685022/src/api/app/controllers/webui/request_controller.rb#L352-L353

Reply:
For reference, this is how we validate the `by_project` and `by_package` parameters now: https://github.com/openSUSE/open-build-service/blob/b741ef6a004afcfc4cfd7086e8db5ebb76685022/src/api/app/models/bs_request_permission_check.rb#L62-L79

Reply:
Good catch, I switched away from strings to user_id/project_id/package_id now.